src/actors/symbolication.rs

```diff
@@ -93,6 +93,13 @@ impl From<&SymbolicationError> for SymbolicationResponse {
     }
 }
 
+pub fn abnormal_session_on_timeout(e: SymbolicationError) -> SymbolicationError {
+    if matches!(e, SymbolicationError::Timeout) {
```

Comment:
According to this mapping those are different:

https://github.com/getsentry/symbolicator/blob/3e4e53416069d343dfa57bf2d92a6e9454f697dc/src/actors/symbolication.rs#L83-L85

Not quite sure how these should map to sessions though?

Reply:
looking at this now I'd expect `Io` and `Canceled` to end up counting as errored sessions I guess?  Is that currently the case?

Reply:
If they log a sentry (error) event, then yes.

Reply:
I had a look and they don't do this at this level.  Instead the toplevel sentry error capturing seems to happen as part of the sentry actix middleware: https://github.com/getsentry/symbolicator/blob/321025a19462e2aec70857a10215b8bfa5585f1d/src/utils/sentry.rs#L204
But IIUC this is outside of the session because you'll have closed it already here before it gets to that middleware?  However you're looking to treat a session as a symbolication task and not as a http request/response pair.  Removing errors from the http request/response seems a little unwise too, so perhaps this code you're adding should also make sure to add top-level error capturing?

Reply:
`Canceled` should probably count as fatal, along with every symbolication error kind other than `Timeout`. It indicates that symbolication failed completely and there is no valid result to the symbolication task.